src/main/webapp/jsp/detail/mining-link-table.hbs

```diff
@@ -0,0 +1,14 @@
+<script id='mining-link-table' type='text/x-handlebars-template'>
+    <section>
```

Comment:
This section should be hidden by default and only shown if we have some hits in the extracted link search